modules/media/jsx/uploadForm.js

```diff
@@ -118,13 +118,14 @@ class MediaUploadForm extends React.Component {
               required={true}
               value={this.state.formData.visitLabel}
             />
-            <SelectElement
+            <DatalistElement
               name="forSite"
               label="Site"
               options={this.state.Data.sites}
+              strictDatalist={true}
               onUserInput={this.setFormData}
               ref="forSite"
-              required={true}
+              required={false}
```

Comment:
i feel like changing site to not be a required field is outside the scope of this PR?